apps/design/backend/src/app.test.ts

```diff
@@ -341,11 +429,6 @@ test('Export all ballots', async () => {
       layoutOptions,
     });
   }
-
-  const electionPackageResult = await apiClient.exportElectionPackage({
-    electionId,
-  });
-  expect(electionHash).toEqual(electionPackageResult.electionHash);
```

Comment:
This checks a fairly important invariant - the exported ballots should match the exported election package. I think we should still test it in some way (and same with test decks below). Maybe a helper function to run through the whole election package export process would be useful here?

Reply:
Ah nice, thanks for pointing that out. Added a test to cover the invariant, as well as the suggested helper: https://github.com/votingworks/vxsuite/pull/4462/commits/f9722c0460e0ed4c06bcd3a5ac85cd365ce4fd11